src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -22,33 +22,40 @@ public class LogsCenter {
     private static final int MAX_FILE_SIZE_IN_BYTES = (int) (Math.pow(2, 20) * 5); // 5MB
     private static final String LOG_FILE = "addressbook.log";
     private static Level currentLogLevel = Level.INFO;
-    private static final Logger logger = LogsCenter.getLogger(LogsCenter.class);
-    private static FileHandler fileHandler;
-    private static ConsoleHandler consoleHandler;
+    // The logger that will be used as the parent of all other loggers created by this class.
+    private static final Logger logger;
+    private static Logger baseLogger;
```

Comment:
```suggestion
    private static Logger baseLogger; // to be used as the parent of all loggers
    private static final Logger logger; // logger for this class
```
Perhaps just end-of-line comments (as suggested above) are enough here?
I put baseLogger above logger because I felt it is more intuitive to learn about baseLogger first. But either order is OK.

Reply:
Any reason why one is final but the other is not?